### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by bernabasy**
In this pull request, the following things are done
     -index.html file is added to finish the mobile version and
    -styel.css  file is added to finish the mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,22 +40,224 @@ <h2 class="contact-title">LET’S CONNECT</h2>
...
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="project-button" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+        </section>
+        <section class="headline-2">
+          <h1 class="title">About Myself</h1>
```
Comment (leonardodiasb) [inline index.html:178]: - [ ] The `h1` tag should be used for the main title of the page and should not be more than 1 per page. The `h1` was used for your name in the first section and an `h2` maybe would be a better fit for the other section titles. Kindly change that.

File: style.css [hunk 2]
```diff
@@ -52,15 +52,27 @@ nav {
...
+
+.headline {
+  background-image: url("image/header-shapes-mobile.png");
+  border-bottom-left-radius: 60px;
+}
+
+.headline-2 {
+  background-image: none;
+  border-bottom-left-radius: 60px;
+}
```
Comment (leonardodiasb) [inline style.css:74]: - [ ] It is a project requirement to use 'flexbox' in the 'about' section. Please use flexbox to align the elements of this section.
 -
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @bernabasy, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - GREAT UI design. Matching perfectly with the Figma template. ✔️
 - No linter errors. ✔️ 
 - Professional PR message. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cwaku — APPROVED** [review]
Hi @bernabasy 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**